content/ui-patterns/forms.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Forms
+---
+
+import {Box, Heading, Link} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Forms are used to complete tasks that require data input from the user. For example: creating a new repo, configuring settings, and logging in.
+</Box>
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Primer's form design guidelines aim to minimize the effort and cognitive load required to complete a task that involves a form.
+</Box>
+
+## Form control anatomy
+
+![diagrams labeling the anatomy of a text field and a checkbox field](https://user-images.githubusercontent.com/2313998/168928812-fa5c9ad7-4f92-4e9d-8e6e-62eb4413985c.png)
```

Comment:
The anatomy image is very blurry on non-high-resolution displays (same for most images on this page in fact). Compare it with the paragraph text in the screenshot:

> ![Screenshot](https://user-images.githubusercontent.com/293280/169879310-48b22366-7858-48aa-8910-bfea1a99cfce.png)

For the ActionList guidelines, I made the screenshots exactly 2x the rendered output. @mperrotti can you make these adjustments?

Reply:
Good catch. These aren't screenshots, they're rendered and exported from Figma. I'll try bumping the export up to @2x

Reply:
For this particular image, what do you think about pulling those pointer lines out a bit instead of overlapping elements? It looks a little busy to me. You might also consider aligning the label text on one side– so for the lefthand labels (Label, Input) they could all align on the right (end). 

I have an example handy from the Autocomplete docs:

<img width="960" alt="Autocomplete anatomy" src="https://user-images.githubusercontent.com/18661030/170736452-b6d47909-eecf-4f8f-ba5f-481aeff647dc.png">

Reply:
@mperrotti the connector lines between the label and the component with the circle on the end- the way the circle overlaps part of the component (or whatever its pointing to) makes it a little hard to see throughout. Just wondering if it might look more clear to add some white space between the circle and the component like in the screenshot of Autocomplete?

![circle overlapping ui element](https://user-images.githubusercontent.com/18661030/171426141-e7e97c54-0dda-4715-b76a-5f6a572aaf51.png)

Reply:
Thanks for clearing that up. I think that would be a nice improvement. Will fix.